trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -279,7 +279,7 @@ The `tracestate` field value is a `list` as defined below. The `list` is a serie
 A simple example of a `list` with two `list-member`s might look like: `vendorname1=opaqueValue1,vendorname2=opaqueValue2`.
 
 ```
-list  = list-member 0*31( OWS "," OWS list-member )
+list  = list-member 0*31("," list-member )
```

Comment:
It is hard to forbid it due to external dependency. Currently we allow multiple tracestate headers, which makes it possible for HTTP frameworks to generate OWS automatically.

For example:
tracestate: A=1
tracestate: B=2

It is possible that an HTTP framework will preprocess the header and give something like **tracestate: A=1, B=2** to the tracecontext library.